gustaf/helpers/raise_if.py

```diff
@@ -33,3 +34,58 @@ def raiser(self):
 
     else:
         return raiser
+
+
+class ModuleImportRaiser():
+    """
+    Class used to have better import error handling in the case that a package
+    package is not installed. This is necessary due to that some packages are
+    not a dependency of `gustaf`, but some parts require them to function.
+    Examples are `splinepy` and `vedo`.
+    """
+
+    def __init__(self, lib_name: str) -> None:
+        self._message = str(
+                "Parts of the requested functionality in gustaf depend on the "
+                f"external `{lib_name}` package which could not be found on "
+                "your system. Please refer to the installation instructions "
+                "for more information."
+        )
+
+    def __call__(self, *args: Any, **kwds: Any) -> Any:
+        """
+        Is called when the object is called by object(). Will notify the user,
+        that the functionality is not accessible and how to proceed to access
+        the functionality.
+        """
+        raise ImportError(self._message)
+
+    def __getattr__(self, __name: str) -> Any:
+        """
+        Is called when any attribute of the object is accessed by object.attr.
+        Will notify the user, that the functionality is not accessible and how
+        to proceed to access the functionality.
+        """
+        if __name == "_LibraryCanNotBeLoadedHelper__message":
```

Comment:
I am not sure, but maybe this?
```suggestion
        if __name == "_ModuleImportRaiser__message":
```